community/samples/serving/helloworld-java-quarkus/README.md

```diff
@@ -0,0 +1,452 @@
+                 </systemProperties>
+               </configuration>
+             </execution>
+           </executions>
+         </plugin>
+         <plugin>
+           <groupId>io.quarkus</groupId>
+           <artifactId>quarkus-maven-plugin</artifactId>
+           <version>${quarkus.version}</version>
+           <executions>
+             <execution>
+               <goals>
+                 <goal>build</goal>
+               </goals>
+             </execution>
+           </executions>
+         </plugin>
+         <plugin>
+           <groupId>org.codehaus.mojo</groupId>
+           <artifactId>build-helper-maven-plugin</artifactId>
+           <version>${build-helper-maven-plugin.version}</version>
+           <executions>
+             <execution>
+               <id>add-resource</id>
+               <phase>generate-resources</phase>
+               <goals>
+                 <goal>add-resource</goal>
+               </goals>
+               <configuration>
+                 <resources>
+                   <resource>
+                     <directory>${project.basedir}/src/main/knative</directory>
+                     <targetPath>${project.build.directory}/knative</targetPath>
+                     <filtering>true</filtering>
+                   </resource>
+                 </resources>
+               </configuration>
+             </execution>
+           </executions>
+         </plugin>
+       </plugins>
+     </build>
+     <dependencies>
+       <dependency>
+         <groupId>io.quarkus</groupId>
+         <artifactId>quarkus-resteasy</artifactId>
+       </dependency>
+       <dependency>
+         <groupId>io.quarkus</groupId>
+         <artifactId>quarkus-junit5</artifactId>
+         <scope>test</scope>
+       </dependency>
+       <dependency>
+         <groupId>io.rest-assured</groupId>
+         <artifactId>rest-assured</artifactId>
+         <scope>test</scope>
+       </dependency>
+     </dependencies>
+     <profiles>
+       <profile>
+         <id>native</id>
+         <build>
+           <plugins>
+             <plugin>
+               <groupId>io.quarkus</groupId>
+               <artifactId>quarkus-maven-plugin</artifactId>
+               <version>${quarkus.version}</version>
+               <executions>
+                 <execution>
+                   <goals>
+                     <goal>native-image</goal>
+                   </goals>
+                   <configuration>
+                     <enableHttpUrlHandler>true</enableHttpUrlHandler>
+                   </configuration>
+                 </execution>
+               </executions>
+             </plugin>
+           </plugins>
+         </build>
+       </profile>
+     </profiles>
+   </project>
+   ```
+
+1. Run the application locally:
+
+   ```shell
+   ./mvnw compile quarkus:dev
+   ```
+
+   Go to `http://localhost:8080/` to see your `Hello World!` message.
+
+1. In your project directory, create a file named `Dockerfile` and copy the code
```

Comment:
Should all of these steps be moved before "locally testing your sample"? We should organize the content into distinct steps/sections so that its clear what the goal/purpose is at each stage in the guide.

Reply:
If the user chooses Knative Build, what would those steps be?

Reply:
removed knative build section. that should prob go into knative build-template separately.